text/0000-remove-stacks-mixins.md

```diff
@@ -0,0 +1,112 @@
+
+App image builds fail if the build image and selected run image have mismatched metadata. We may introduce flags or additional labels to skip this validation (e.g., for cross-compilation or minimal runtime base images). An image without a specified Distribution is compatible with images specifying any Distribution. An image specifying a Distribution without a Version is compatible with images specifying any Versions of that Distribution.
+
+When an app image is rebased, `pack rebase` will fail if the new run image and previous run image have mismatched metadata. This check may be skipped for Distribution and Version by passing a new `--force` flag to `pack rebase`.
+
+#### Example: buildpack.toml `platforms` table
+
+```toml
+[[platforms]]
+os = "linux"
+arch = "x86_64"
+[[platforms.distros]]
+name = "ubuntu"
+[[platforms.distros.versions]]
+version = "18.04"
+packages = ["pkg:deb/ubuntu/curl"]
+[[platforms.distros.versions]]
+version = "20.04"
+packages = ["pkg:deb/ubuntu/curl2"]
+
+[[platforms]]
+os = "linux"
+arch = "x86_64"
+[[platforms.distros]]
+name = "ubuntu"
+packages = ["pkg:deb/ubuntu/curl"]
+[[platforms.distros.versions]]
+version = "14.04"
+[[platforms.distros.versions]]
+version = "16.04"
+```
+
+## Mixins
+
+The mixins label on each base image is replaced by a layer in each base image containing a single file consisting of a [CycloneDX](https://cyclonedx.org)-formatted list of packages. Each package entry has a [PURL](https://github.com/package-url/purl-spec)-formatted ID that uniquely identifies the package.
+
+### Validations
+
+Buildpack base image metadata and packages specified in `buildpack.toml`'s `platforms` list are validated against the runtime and build-time base images.
+
+Runtime and build-time base image packages are no longer validated against each other.
+
+When an app image is rebased, `pack rebase` will fail if packages are removed from the new runtime base image. This check may be skipped by passing a new `--force` flag to `pack rebase`.
+
+## Runtime Base Image Requirements
+
+Buildpacks may specify a list of package names (as PURL URLs without versions or qualifiers) in a `packages` table in the build plan. The build will fail if these packages are not present in the runtime base image.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- Involves breaking changes.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Keep stacks.
+- Make the above changes but keep some existing terminology: stacks, build-image, run-image.
+- Continue to allow for static definition of runtime package requirements in `buildpack.toml`.
+
+# Unresolved Questions
```

Comment:
How does this change the `[stack]` in `builder.toml`?

Reply:
Added to unresolved questions